internal/venice-common/src/main/java/com/linkedin/venice/views/ViewUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.venice.views;
+
+import com.linkedin.venice.meta.Store;
+import com.linkedin.venice.utils.ReflectUtils;
+import com.linkedin.venice.utils.VeniceProperties;
+import org.apache.avro.Schema;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class ViewUtils {
+  private static final Logger LOGGER = LogManager.getLogger(ViewUtils.class);
+
+  public static VeniceView getVenicePartitioner(String viewClass, VeniceProperties params, Store veniceStore) {
```

Comment:
Nit: maybe change the name to `getVeniceView` :-)

Reply:
doh